27.05.2020 16:00:39.877 - app - ERROR - Can not create user: averjan_63, err: (pymysql.err.DataError) (1406, "Data too long for column 'email' at row 1")
[parameters: {'username': 'averjan_63', 'password': '+yHIRD*7T2', 'email': 'elise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmai ... (1902 characters truncated) ... _2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.comelise_2000@hotmail.com', 'access': '1', 'active': '0', 'start_active_time': None}]
 [27/May/2020 16:00:39] "[32mPOST /login HTTP/1.1[0m" 302 -
172.18.0.1 - - [27/May/2020 16:00:39] "[33mGET /api/del_user/artemi04 HTTP/1.1[0m" 404 -
172.18.0.1 - - [27/May/2020 16:00:39] "[37mGET /api/del_user/kabanovanonna HTTP/1.1[0m" 204 -
172.18.0.1 - - [27/May/2020 16:00:39] "[33mGET /api/block_user/medvedevmihe HTTP/1.1[0m" 404 -
172.18.0.1 - - [27/May/2020 16:00:39] "[37mGET /api/block_user/selivan_63 HTTP/1.1[0m" 200 -
